packages/hoppscotch-common/src/helpers/editor/themes/baseTheme.ts

```diff
@@ -63,6 +63,8 @@ export const baseTheme = EditorView.theme({
   ".cm-panels": {
     backgroundColor: "var(--primary-light-color)",
     color: "var(--secondary-light-color)",
+    zIndex: "1",
+    top: "110px !important",
```

Comment:
Considering the offset of the nearest scrolling ancestor.
<img width="1151" alt="Screenshot 2023-04-17 at 12 34 38" src="https://user-images.githubusercontent.com/1501599/232537508-77b9034d-9604-499d-b374-9390e378c2a5.png">